setup_ch4_inversion.sh

```diff
@@ -463,8 +468,8 @@ if  "$SetupSpinupRun"; then
     sed -i -e "s|${StartDate}|${SpinupStart}|g" \
            -e "s|${EndDate}|${SpinupEnd}|g" \
            -e "s|Do analytical inversion?: T|Do analytical inversion?: F|g" \
-           -e "s|{PERTURBATION}|1.0|g" \
-           -e "s|{ELEMENT}|0|g" input.geos
+           -e "s|pertpert|1.0|g" \
+           -e "s|clustnumclustnum|0|g" input.geos
```

Comment:
Currently {PERTURBATION} and {ELEMENT} are not present in the template input.geos.CH4 file, so I reverted this back to the original sed command. 

However, it seems like we should probably change the input.geos.CH4 file to have {PERTURBATION} and {ELEMENT} instead of pertpert and clustnumclustnum, but this would require updating the GCClassic code and updating to a version with that update. Is this correct @msulprizio?

Reply:
I have those locally but have not pushed those changes to our GEOS-Chem fork. I will do so ASAP.

Reply:
Great, thanks!